addons/udes_common/models/models.py

```diff
@@ -134,3 +135,43 @@ def selection_display_name(self, selection_field_name):
     # Odoo has a nice function that handles all the weird ways you can define selections!
     selection_values_dict = dict(selection_field._description_selection(self.env))
     return selection_values_dict.get(self[selection_field_name], False)
+
+
+class Base(models.AbstractModel):
+    _inherit = 'base'
+
+    @api.model
+    def fields_view_get(self, view_id=None, view_type='form', toolbar=False, submenu=False):
```

Comment:
Can we add doc-strings to all the methods?

Reply:
Added the docstring in this method, the others i think already had